### ISSUE TITLE
Hexadecimal is converted into number in action parameter

### ISSUE THREAD

**Issue by andylim0221** [moleculerjs/moleculer#721]
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository

## Current Behavior

We have an action with the following parameter setup:

```
params: {
  transactionHash: 'string'
}
```

We pass a hexadecimal string, eg. '0xab706...' into the params, the hexadecimal string is being converted into number '7.7544153918976e+76'.

## Expected Behavior

Expected to pass the original hexadecimal string and not convert it.

### Context

* Moleculer version:  ^0.13.0
* NodeJS version: 10.16.0
* Operating System: MacOS Catalina

**Comment by icebob** [comment]
I don't think so.
https://repl.it/@icebob/moleculer-dev-hexdec

**Comment by AndreMaz** [comment]
I cloud reproduce with REPL. So it's not a validation issue

![image](https://user-images.githubusercontent.com/9802754/79497833-48635680-8020-11ea-9239-81c9bcbdf484.png)

**Comment by icebob** [comment]
In this case, the problem is in vorpal?

**Comment by AndreMaz** [comment]
Yes, I think it is. @andylim0221 can you provide more info?  Did you use REPL?

**Comment by andylim0221** [comment]
@AndreMaz I was using moleculer-repl.

**Comment by andylim0221** [comment]
however, when i query it through the url, it works out as what I expected. Perhaps there must be something wrong with the moleculer-repl.
